src/current/v23.2/create-virtual-cluster.md

```diff
@@ -0,0 +1,110 @@
+
+`CREATE VIRTUAL CLUSTER` requires one of the following privileges:
+
+- The `admin` role.
+- The `MANAGEVIRTUALCLUSTER` [system privilege]({% link {{ page.version.version }}/security-reference/authorization.md %}#privileges) allows the user to run all the related `VIRTUAL CLUSTER` SQL statements for physical cluster replication.
+- The `REPLICATION` [system privilege]({% link {{ page.version.version }}/security-reference/authorization.md %}#privileges) allows a user to connect to stream data out of a virtual cluster for physical cluster replication.
+
+Use the [`GRANT SYSTEM`]({% link {{ page.version.version }}/grant.md %}) statement:
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+GRANT SYSTEM MANAGEVIRTUALCLUSTER TO user;
+~~~
+
+## Synopsis
+
+<div>
+{% include {{ page.version.version }}/physical-replication/create-virtual-cluster-diagram.html %}
+</div>
+
+## Parameters
+
+Parameter | Description
+----------+------------
+`virtual_cluster_name` | The name for the new virtual cluster.
+`LIKE virtual_cluster_spec` | Ensures the virtual cluster on the standby is created with the correct [capabilities](#capabilities).
+`primary_virtual_cluster` | The name of the primary's virtual cluster to replicate.
+`primary_connection_string` | The PostgreSQL connection string to the primary cluster. Refer to [Connection string](#connection-string) for more detail.
+`replication_options_list`| Options to modify the replication streams. Refer to [Options](#options).
+
+## Options
+
+Option | Description
+-------+-------------
+`RETENTION` | Configure a [retention window]({% link {{ page.version.version }}/physical-cluster-replication-technical-overview.md %}#cutover-and-promotion-process) that will control how far in the past you can [cut over]({% link {{ page.version.version }}/cutover-replication.md %}) to.
+
+## Connection string
+
+When you [initiate a replication stream]({% link {{ page.version.version }}/set-up-physical-cluster-replication.md %}#step-4-start-replication) from the standby cluster, it is necessary to pass a connection string to the primary cluster:
+
+{% include_cached copy-clipboard.html %}
+~~~
+'postgresql://{replication user}:{password}@{node IP or hostname}:26257/?options=-ccluster=system&sslmode=verify-full&sslrootcert=certs/{primary cert}.crt'
+~~~
+
+To form the connection string to the primary cluster, use the following query parameters:
+
+Query parameter | Description
+----------------+------------
+`replication user` | The user on the primary cluster that has the `REPLICATION` system privilege. Refer to the [Create a replication user and password]({% link {{ page.version.version }}/set-up-physical-cluster-replication.md %}#create-a-replication-user-and-password) for more detail.
+`password` | The replication user's password.
+`node ID or hostname` | The node IP address or hostname of any node from the primary cluster.
+`ccluster=system` | The parameter to connect to the system interface on the primary cluster.
```

Comment:
This is different from the others as it isn't something you need to _replace_ in the query above.

Reply:
I have changed up this section with new introductory sentences and titles in the table. I think it is worth including them all (I had missed some — not sure how) so there is some description. I could see needing to link to this at some point.